CLI.md

```diff
@@ -794,9 +798,34 @@ Initialize a wallet from the genesis configuration
 
 
 
+## `linera wallet request-chain`
+
+Request a chain from a faucet and add it to the wallet
```

Comment:
I'd be more precise here - we're requesting _creation_ of a new chain from the wallet.